micro-infra-spring-config/src/main/java/com/ofg/infrastructure/property/AppCoordinates.java

```diff
@@ -39,8 +41,29 @@ public static AppCoordinates defaults(Resource microserviceConfigResource) {
         this.countryCode = requireNonNull(countryCode);
     }
 
-    public File getConfigFolder(File rootFolder) {
+    public ConfigLocations getConfigLocations(File rootFolder) {
+        return new ConfigLocations(
+                getCommonConfigFolder(rootFolder),
+                getEnvConfigFolder(rootFolder),
+                getCountryConfigFolder(rootFolder)
+        );
+    }
+
+    private File getCommonConfigFolder(File rootFolder) {
+        File folder = new File(rootFolder, COMMON_DIR);
+        return addNameComponentsToFolder(folder);
+    }
+
+    private File getEnvConfigFolder(File rootFolder) {
         File folder = new File(rootFolder, environment);
+        return addNameComponentsToFolder(folder);
+    }
+
+    private File getCountryConfigFolder(File rootFolder) {
+        return new File(getEnvConfigFolder(rootFolder), countryCode);
+    }
+
+    private File addNameComponentsToFolder(File folder) {
         final String[] components = nameComponents();
         for (int i = 0; i < components.length - 1; i++) {
```

Comment:
It seems that when we have an applicaion name like /com/ofg/micro-app then a configuration directory will be for example /properties/prod/com/ofg but not /properties/prod/com/ofg/micro-app. Inside this directory we will have micro-app.properties and micro-app.yaml file.

It is consistent with docs: https://github.com/4finance/micro-infra-spring/wiki/Centralized-configuration-management.

Nevertheless some comment in the code will be welcome as it is indeed not obvious.